### PR TITLE
docs(callbox): variants and docs - vue3

### PR DESCRIPTION
**Pull request by juliodialpad**
# Vue 3 version of #1150

### CODE REVIEW

File: recipes/leftbar/callbox/callbox.stories.js [hunk 1]
```diff
@@ -44,6 +45,12 @@ const DefaultTemplate = (args) => createTemplateFromVueFile(
   DtRecipeCallboxDefaultTemplate,
 );
 
+const VariantsTemplate = (args, { argTypes }) => createTemplateFromVueFile(
+  args,
+  argTypes,
```
Comment (iropolo) [inline recipes/leftbar/callbox/callbox.stories.js]: Seems like it is not working cos this `argTypes`, remove it and it will works.

**Review by iropolo — CHANGES_REQUESTED** [review]
Variants not working

File: recipes/leftbar/callbox/callbox.stories.js (hunk 1 above)
Comment (juliodialpad) [reply recipes/leftbar/callbox/callbox.stories.js]: Thanks for catching that! forgot to remove it while migrating to vue3

**Comment by juliodialpad** [comment]
Renamed the constants to avoid conflicts

**Review by iropolo — APPROVED** [review]
Looks goods, thanks for the update

**Pull request merged**